Client.xcodeproj/project.pbxproj

```diff
@@ -8241,25 +8246,24 @@
 			};
 			name = Beta;
 		};
-		27F443A02135E11200296C58 /* Debug */ = {
+		27A1AC1624884F7A00344503 /* Enterprise */ = {
 			isa = XCBuildConfiguration;
+			baseConfigurationReference = 27A1AC1424884EB300344503 /* Enterprise.xcconfig */;
 			buildSettings = {
-				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = NO;
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ALWAYS_SEARCH_USER_PATHS = NO;
-				CLANG_ANALYZER_NONNULL = YES;
-				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
-				CLANG_CXX_LANGUAGE_STANDARD = "gnu++14";
+				CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES = YES;
+				CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED = YES;
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++0x";
```

Comment:
We shouldn't use `gnu++0x` Use the latest C++. `0x` is beta (working name) for C++11.

Reply:
This is set to c++11 in the UI and is set the same for all of them: 
<img width="486" alt="image" src="https://user-images.githubusercontent.com/529104/88209418-af486b00-cc20-11ea-849e-5e8f322213bd.png">